third_party/rust/third_party.toml

```diff
@@ -201,8 +204,8 @@ if (target_cpu == "arm64" &&
 '''
 
 [dependencies.curve25519-dalek]
-version = "3.2.1"
-features = [ "alloc", "serde", "std", "u64_backend" ]
```

Comment:
I think this actually isn't needed anymore for v3 or v4 because it's no longer a direct dep here

Reply:
don't we need it to define `gn-variables-lib`?